test/e2e-go/features/catchup/basicCatchup_test.go

```diff
@@ -164,6 +166,7 @@ func TestStoppedCatchupOnUnsupported(t *testing.T) {
 	a.NoError(err)
 	cloneClient, err := fixture.StartNode(cloneDataDir)
 	a.NoError(err)
+	defer shutdownClonedNode(cloneDataDir, &fixture, t)	
```

Comment:
looks to me that a better way to do that would be to have fixture.StartNode detect that the given node directory isn't any of the ones it's currently managing, and add it internally.
Then, when fixture.Shutdown is called, you would add it there as well.

The rationale here is that it would save us from duplicating the shutDownClonedNode you've added here.

Reply:
The way things are organized right now makes it difficult to unify the stopping and cleanup of the nodes. 

libgoalFixture.Shutdown calls netdeploy.Network Stop to stop the network. 
netdeploy.Network uses the data structures populated from netdeploy.CreateNetworkFromTemplate to identity which nodes to stop. 

Clone is done from nodecontrol package, outside of netdeploy. Given that the nodes are generated using separate packages, makes unification of their cleanup difficult. 

As we discussed, the alternative is to not clone, but to start the network nodes individually. This is a more promising approach for later.